recipes/dis3/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "dis3" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "ee9f71e1a0c01a3ba742a36be91147948db99da3ee29094dccb48857c54608ef" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py3k]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - dis3
+
+about:
+  home: https://pypi.python.org/pypi/dis3
```

Comment:
Technically we might want to list the old repo location and note it is defunct, but the license file comment seems to cover that already.

Reply:
note: Done below.

Thanks for the follow-up. Looks good.